harness/features/variable.cloud.test.ts

```diff
@@ -0,0 +1,260 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createClient, 
+    createUser,
+    wait,
+    callVariable, 
+    forEachVariableType,
+    variablesForTypes
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import nock from 'nock'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+describe('Variable Tests - Cloud', () => {
+    forEachSDK((name) => {
+        let url: string
+        const mockServerUrl 
```

Comment:
this should be the base url of the mock server, not the proxy server, so it'll redirect to `${mockServerUrl}/v1/variables/key`

Reply:
yea I just thought the mock server was going to use the same format so that every mock is done by client Id as well. It might not actually be necessary but I think thats what the others are doing in their tests